extension/src/standalone_pages/InstructorDashboardRenderVideoInfo.js

```diff
@@ -194,6 +254,44 @@ function renderPassiveFeedback(video, response) {
             }
         }
     });
+
+    let canvas = document.getElementById(`passiveFeedback_${video.videoID}`);
+    canvas.onclick = function(evt) {
+        let firstPoint = chart.getElementAtEvent(evt)[0];
+
+        if (firstPoint) {
+            let label = chart.data.labels[firstPoint._index];
+            console.debug(label, keywordsMap[label]);
+            if (!keywordsMap[label]) {
+                getVideoContextKeyWords(video, label);
+            }
+            displayContextKeyWords(label);
+        }
+    };
+}
+
+async function getVideoContextKeyWords(video, timestamp) {
+    chrome.runtime.sendMessage({
+        type: "FetchVideoContextKeyWords",
+        videoID: video.videoID,
+        timestamp: timestamp,
+        duration: video.video_duration
+    }, (response) => {
+        keywordsMap[timestamp] = processContextKeyWords(response);
+        if (Swal.isVisible()) {
+            Swal.close();
+        }
+        displayContextKeyWords(timestamp);
+    });
+}
+
+function processContextKeyWords(result) {
+    if (result.status) {
+        return { rake: result.data.rake, bert: result.data.bert };
+    } else {
+        
+        return "No Context words at this timestamp";
```

Comment:
This is another mix use of values type. Isn't this 100% going to break the code in the display function?

I would recommend that you follow the format above (line 290) and hardcode the values with `"No Context words at this timestamp";`